crates/common/tedge_config/src/tedge_config_cli/error.rs

```diff
@@ -29,6 +31,9 @@ pub enum TEdgeConfigError {
 
     #[error(transparent)]
     DirNotFound(#[from] tedge_utils::paths::PathsError),
+
+    #[error(transparent)]
+    ConfigNotSet(#[from] ConfigNotSet),
```

Comment:
I guess it is added because of the mistake in `software_manager/config.rs`.

Reply:
right, fixed it.